ntoskrnl/kd/wrappers/gdbstub_powerpc.c

```diff
@@ -1344,11 +1344,13 @@ KdpGdbEnterDebuggerException(PEXCEPTION_RECORD ExceptionRecord,
             case '?':
               /* a little hack to send more complete status information */
               goto stop_reply;
+/*
```

Comment:
Can you #if 0 instead, as that's disabled code.

Reply:
Done: `#if 1 else` even.